src/redux/categories/categories.js

```diff
@@ -0,0 +1,25 @@
+const UNDER_CONSTRUCTION = 'UNDER_CONSTRACTION';
+const COMPELET = 'COMPLETE';
+
+const categories = [];
+
+export const unCompelet = () => ({
+  type: UNDER_CONSTRUCTION,
+});
+
+export const complete = () => ({
+  type: COMPELET,
+});
+
+const categoriesReducer = (state = categories, action) => {
+  switch (action.type) {
+    case UNDER_CONSTRUCTION:
+      return false;
+    case COMPELET:
```

Comment:
- [ ] This module requires that if action is of type COMPLETE (which in your case represents CHECK_STATUS), then the function should return a string `'Under construction'` else if action type is different, it should return the state with no change